commander/src/bootstrapping/templates/lisk-template-ts/templates/plugin/src/app/plugins/plugin.ts

```diff
@@ -4,25 +4,14 @@ import type { BaseChannel } from 'lisk-sdk';
  /* eslint-disable class-methods-use-this */
  /* eslint-disable  @typescript-eslint/no-empty-function */
  export class <%= className %> extends BasePlugin {
-	// private _channel!: BaseChannel;
 
 	public name: '<%= name %>';
 
 	public get nodeModulePath(): string {
-		return <%= __filename %>;
+		return  __filename;
 	}
 
-	public get events(): string[] {
-		return [
-			// 'block:created',
-			// 'block:missed'
-		];
-	}
-
-	public async load(_: BaseChannel): Promise<void> {
-		// this._channel = channel;
-		// this._channel.once('app:ready', () => {});
-	}
+	public async load(_: BaseChannel): Promise<void> {}
```

Comment:
```suggestion
	public async load(): Promise<void> {}
```
Looking at `BasePlugin` we should also remove the input and the import